src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/PreferIsKindAnalyzer.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers
     [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
     public sealed class PreferIsKindAnalyzer : DiagnosticAnalyzer
     {
-        internal static DiagnosticDescriptor Rule = new(
+        internal static readonly DiagnosticDescriptor Rule = new(
```

Comment:
Have you made the changes manually? It might be better to enable a rule for such cases.

Reply:
@Evangelink Yeah these were manual changes :(

There is an analyzer, and it looks like it should be enabled as a warning. But I don't know why it's not working:

https://github.com/dotnet/roslyn-analyzers/blob/90089d9884b3feb90ef33cfe267dabd3c2854768/.editorconfig#L63

Reply:
@Youssef1313 That rule only works on private fields. There is no reliable way to flag internal fields without analyzing the entire solution.